content/docs/examples/advanced-gateways/ingress-certmgr/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Securing Kubernetes Ingress with Cert-Manager
+description: Demonstrates how to obtain "Let's Encrypt" TLS certificates for Kubernetes Ingress automatically using Cert-Manager.
+weight: 70
+keywords: [traffic-management,ingress,https,cert-manager,acme,sds]
+---
+
```

Comment:
One thing that is missing for me in the tutorial is why I should choose this support over an nginx based ingress controller if my cloud provider k8s already provides that.

What would be the advantages?  It doesn't seem or make sense that I can use virtual service on the ingress so why would anyone use this over nginx based ingress controller if their cloud provider already provides one within their hosted k8s env. 

Maybe telemetry is one advantage?

Reply:
Elaborated on this in the intro a little bit, but could definitely use some help of tech writer here later on.